near/omni-bridge/src/lib.rs

```diff
@@ -495,14 +504,19 @@ impl Contract {
             "Unknown factory"
         );
 
+        let decimals = self.token_decimals.get(&init_transfer.token);
```

Comment:
Maybe it makes sense to return an error if token_decimal is not found?

Reply:
But not all tokens have the decimal.
Do you propose also to insert the decimal in the `deploy_token_internal`? In this case, we should insert decimal with same value for `decimal` and `origin_decimal`